jstests/oidc/lib/oidc_fixture.js

```diff
@@ -306,21 +307,22 @@ export class OIDCFixture {
             this.idps[idp_url].start();
         }
 
-        this.cluster = new clusterClass(this.oidc_providers, this.audit_path, jwksMinimumQuiescePeriodSecs);
+        this.cluster =
+            new clusterClass(this.oidc_providers, this.audit_path, jwksMinimumQuiescePeriodSecs);
         this.admin_conn = this.cluster.connection();
         this.admin = this.admin_conn.getDB("admin");
         assert.commandWorked(this.admin.runCommand(
-            { createUser: "admin", pwd: "admin", roles: [{ role: "root", db: "admin" }] }));
+            {createUser: "admin", pwd: "admin", roles: [{role: "root", db: "admin"}]}));
         assert(this.admin.auth("admin", "admin"));
 
-        var callback = function (authInfo) {
+        var callback = function(authInfo) {
             print("OIDCFixture.OIDCIdPAuthCallback: ", JSON.stringify(this));
             if (this.client_callback) {
-                var issuer = authInfo.activationEndpoint.replace("/device/verify", "")
-                this.client_callback({
-                    user: authInfo.userName,
-                    issuer: issuer,
-                })
+                var issuer =
+                    authInfo.activationEndpoint.replace("/device/verify", "") this.client_callback({
```

Comment:
Absence of semicolon after the call to `replace` seems to confuse the formatter.